sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/persistence/DatabasePersistenceAndroidTest.java

```diff
@@ -785,7 +786,7 @@ public void deleteLogsForGroup() throws PersistenceException {
     }
 
     @Test
-    public void getLogs() throws PersistenceException {
+    public void getLogsWithNormalPriority() throws PersistenceException {
```

Comment:
Does this name change make sense?  There's no equivalent test that can be done for critical logs, is there?  If that's the case I would suggest just using `getAllLogs`

Reply:
The name change was because of getLogsWithMixedPriorities which is a variant of this one.